cli/RELEASING.md

```diff
@@ -0,0 +1,41 @@
+Releasing Cook Executor
+=======================
+
+Cook CLI is released on [PyPI](https://pypi.org/project/cook-executor/)
+
+Prerequisites
+-------------
+Ensure you can build the cli following the instructions in README.md
+
+Install `twine`:
+```bash
+pip3 install twine
+```
+
+Update version number
+------------
+Look at setup.py, and edit the version number which may reference another python variable to be edited.
+
+Test Release
+------------
+Since PyPI does not allow modifying releases, it can be useful to test a release using their test instance.
+```bash
+rm -rf dist/*
+python3 setup.py sdist bdist_wheel
+python3 -m twine upload --repository-url https://test.pypi.org/legacy/ dist/*
+```
+Then, in a separate virtualenv for testing:
+```bash
+pip3 install  --index-url https://test.pypi.org/simple/ --no-deps cook-executor==$VERSION
+pip3 install # install any other required dependencies from the main pypi repo
```

Comment:
This part was confusing to me. I ended up installing cook-client 
`pip3 install cook-client` 
and then using the full path to the cs in my virtual env. Is that what you meant?